src/pages/Cost/Cost.js

```diff
@@ -1,12 +1,353 @@
-import React from "react";
+import React, { useEffect, useState } from 'react';
 import './Cost.scss';
 
 const Cost = () => {
+  const [itemList, setItemList] = useState([]);
+  const [user, setUser] = useState([]);
+  const [checkInput, setCheckInput] = useState('');
+  const [usePoint, setUsePoint] = useState();
+
+  // 장바구니 불러오는 함수
+  const getCartList = () => {
+    fetch(`http://10.58.52.52:8000/products/cart`, {
+      method: 'GET',
+      headers: {
+        Authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwiZW1haWwiOiJkbGdvYWxzMzM5NkBnbWFpbC5jb20iLCJpYXQiOjE2OTg2NTI3MDN9.vrPN3gvnepPB9LE0doiwuv5idJDvWQIK6jzuInjcBr0',
+      },
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        setItemList(data.message);
+      });
+  };
+
+  // 장바구니 불러오기
+  useEffect(() => {
+    getCartList();
+  }, []);
+
+  // 회원정보 불러오기
+  useEffect(() => {
+    fetch(`http://10.58.52.200:3000/products/costUser`, {
+      method: 'POST',
+      headers: {
+        Authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwiZW1haWwiOiJkbGdvYWxzMzM5NkBnbWFpbC5jb20iLCJpYXQiOjE2OTg3MTk0OTJ9.raeSnK2Ql4LCAqVVXd53p2o933AtBHhUmLcLkFDHKP0',
+      },
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        setUser(data.message[0]);
+      });
+  }, []);
+
+  // 상품 개수 추가
+  const plusCount = (id) => {
+    const productQuantity = itemList.find((item) => item.id === id).quantity;
+    const productCount = itemList.find((item) => item.id === id).count;
+
+    if (productCount >= productQuantity)
+      return alert('재고 수량이 부족합니다.');
+
+    fetch(`http://10.58.52.52:8000/products/cart/${id}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MiwiZW1haWwiOiJkbGdvYWxzMzM5NkBnbWFpbC5jb20iLCJpYXQiOjE2OTg2NTI3MDN9.vrPN3gvnepPB9LE0doiwuv5idJDvWQIK6jzuInjcBr0',
+      },
+      body: JSON.stringify({
+        count: itemList.find((item) => item.id === id).count + 1,
```

Comment:
위에서 선언한 productCount를 활용할 수 있지 않을까요?
```suggestion
        count: productCount + 1,
```

Reply:
추후에 변경한거라 확인을 못했습니다!! 수정했습니다!!